shared/utils/count_oval_objects.py

```diff
@@ -111,7 +111,13 @@ def main():
         rule_id = rule.attrib['id']
         oval_refs = []
         for ref in rule.findall(".//check-content-ref"):
-            oval_name = ref.attrib['name']
+            # Skip remotely referenced OVAL checks since they won't have the
+            # 'name' attribute set (just 'href' would be set in that case)
+            try:
+                oval_name = ref.attrib['name']
+            except KeyError:
+                print "\nInfo: Skipping remotely referenced OVAL:"
```

Comment:
I would prefer if we made sure the OVAL has a "href" attribute in this case. Like an assertion check. That way we are sure we are printing the right message.